m/base.go

```diff
@@ -172,11 +177,34 @@ func GetLink(address int) Link {
 // Retrieve a list of units that match specified conditions
 //
 // Conditions are combined using an `and` operation
-func Radar(from string, target1 RadarTarget, target2 RadarTarget, target3 RadarTarget, sortOrder bool, sort RadarSort) Unit {
+func Radar(from Building, target1 RadarTarget, target2 RadarTarget, target3 RadarTarget, sortOrder bool, sort RadarSort) Unit {
 	return nil
 }
 
 // Extract information indicated by sense from the provided block
-func Sensor(block interface{}, sense string) float64 {
+func Sensor(block HealthC, sense string) float64 {
 	return 0
 }
+
+func CreateSensorFuncTranslation(attribute string) transpiler.Translator {
```

Comment:
Please convert this to a private function, as it shouldn't be called outside the package

Reply:
Oh, my mistake. I forgot to change the function to a private function after I moved it. I changed it immediately.:)